.github/workflows/cla.yml

```diff
@@ -0,0 +1,43 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+# explicitly configure permissions, in case your GITHUB_TOKEN workflow permissions are set to read-only in repository settings
+permissions:
+  actions: write
+  contents: read
+  pull-requests: write
+  statuses: write
+
+jobs:
+  CLAAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        uses: contributor-assistant/github-action@v2.4.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.CLA_TOKEN }}
+        with:
+          path-to-signatures: 'signatures/version1/signatures.json'
+          path-to-document: 'https://gist.github.com/af3262c2eb70b14b8e96260e8acfc2cf' # CLA document
+          # branch should not be protected
+          branch: 'main'
+          allowlist: “materialize-bot,materialize-bot-monitoring,dependabot[bot]”
```

Comment:
```suggestion
          allowlist: "materialize-bot,materialize-bot-monitoring,dependabot[bot]"
```

Reply:
Done.

Reply:
Doesn't seem to be done.

Reply:
It was done in https://github.com/MaterializeInc/materialize/pull/28682/commits/440245d28de37d9f3811b1a61b5e959e8e0a2be7 but reverted for whatever reason in https://github.com/MaterializeInc/materialize/pull/28682/commits/03c29dcf91caa1a16b202a5705a94192b05a51ea.